src/main/java/seedu/address/model/person/Person.java

```diff
@@ -25,20 +25,22 @@
     private final Address address;
     private final Set<Tag> tags = new HashSet<>();
     private final Remark remark;
+    private final Status status;
     private Set<Position> positions = new HashSet<>();
 
     /**
-     * Every field must be present and not null.
+     * Every field must be present and not null, except status. If status is null, defaults to applied
      */
-    public Person(Name name, Phone phone, Email email, Address address, Remark remark, Set<Tag> tags,
+    public Person(Name name, Phone phone, Email email, Address address, Remark remark, Set<Tag> tags, Status status,
                   Set<Position> positions) {
-        requireAllNonNull(name, phone, email, address, tags, positions);
+        requireAllNonNull(name, phone, email, address, tags, status, positions);
```

Comment:
Header comment says status could be null, while here we require status to be non-null anyways. Would it be good if we just require status to be non-null, just like the rest?

Reply:
Status being optional is implemented in the parser, when creating an person, a Applied status will be passed in, will change the header comment